chains/1329/assetlist.json

```diff
@@ -53,7 +53,8 @@
         {
             "asset_type": "erc20",
             "erc20_contract_address": "0x0555E30da8f98308EdB960aa94C0Db47230d2B9c",
-            "coingecko_id": "wrapped-bitcoin"
+            "coingecko_id": "wrapped-bitcoin",
+            "recommended_symbol": "WBTC.lz"
```

Comment:
We should rename the wormhole one instead.

Reasoning:
1. we have a handful of assets already named with .wh for wormhole, and much less assets for .lz for layerzero
2. Sei wants to consider the layerzero one as the main WBTC, so we should respect that name and make the other one hard to find (the wormhole asset)